CveXplore/database/maintenance/Sources_process.py

```diff
@@ -1094,6 +1094,80 @@ def populate(self, **kwargs):
         return self.update()
 
 
+class EPSSDownloads(CSVFileHandler):
+    def __init__(self):
+        self.feed_type = "EPSS"
+        self.delimiter = ","
+        super().__init__(self.feed_type, self.delimiter)
+
+        self.feed_url = Configuration.getFeedURL(self.feed_type.lower())
+        self.logger = logging.getLogger(__name__)
+        self.is_update = True
+
+    @staticmethod
+    def process_epss_item(item=None):
+        if item is None:
+            return None
+
+        epss = {
+            "id": item[0],
+            "epss": item[1],
+            "percentile": item[2]
+        }
```

Comment:
Looking at the data that is imported; why would you go for a separate collection? Looking at the rest of the code it looks like there is a one-to-one match between the CVE's and the EPSS, wouldn't it make sense to store the EPSS information straight into the cves collection?

Reply:
You are correct. When I examined the code, I realized that I could implement it similarly to the via4 implementation. I didn't consider checking the data from via4, and that's my mistake. I will make the necessary changes to store the EPSS information directly into the CVEs collection.